docs/volunteering/writing/editing-bobadocs.md

```diff
@@ -0,0 +1,158 @@
+
+**If you already have a fork of BobaDocs, replace this section with "Keeping your repo updated" in the [Appendix](./editing-bobadocs-appendix).**
+
+The files that this website runs off of (known collectively as the site's `repo`) are only allowed to be edited by the BobaLord: this prevents random people from showing up and screwing up the site. This also means that in order to make any changes, you'll need your own copy of the files, known as a `fork`.
+
+First, **make sure you're logged in to GitHub.** Once you're logged in, navigate to the main repo for the site at [https://github.com/essential-randomness/bobadocs](https://github.com/essential-randomness/bobadocs). You can also get there by clicking the "GitHub" link on the top right corner of the site.
+
+Once there, on the top right corner of the GitHub page, you should see a `Fork` button. Click it. You will be automatically redirected to your fork.
+
+![a cropped screenshot of the bobadocs github page, with a red arrow pointing to the fork button](/img/volunteer/bobadocsfork.jpg)
+
+### Branching
+
+In git, a `branch` is a separate version of a repo that can be edited without affecting the main repo. It lets developers separate what they're working on, so problems that arise when working on one issue don't affect any other work that may be being done concurrently. Making good use of branches is key to keeping a project history easy to understand. It also keeps multiple people working on the same project from stepping on each other's toes.
+
+To create a new branch, click the dropdown labelled "main" on the left side of the screen, enter a short, descriptive name for your repo in the textbox, and click "Create branch".
+
+![a cropped screenshot of the bobadocs github page, with red arrows pointing to relevant spots on the branch dropdown](/img/volunteer/bobadocsbranch.jpg)
+
+## Step 1: Navigating to GitHub's editor
+
+### Method 1: Navigate from within your GitHub repo
+
+If you look at your repo, you'll notice that it looks a lot like a filesystem. That's because it is a filesystem! And you can navigate to the different files in it - that is, the *pages in the site* - by clicking through it.
+
+To do that, we need to know a little bit about how Docusaurus is laid out. Docusaurus separates site content into four folders:
+
+- `blog`: containing blogposts (this is unused by BobaDocs at this time)
+- `docs`: containing the pages on the site **(this is the one we want!)**
+- `src`: containing information on how to build the site, including styling
+- `static`: containing static site data, such as images
+
+We want to edit a page in the site, so click `docs`.
+
+![a screenshot of the git repo, with a red arrow pointing to the docs folder](/img/volunteer/bobadocsnav1.jpg)
+
+You'll notice four new folders: `engineering`, `product`, `users`, and `volunteer`. Each of these folders corresponds to one of the guides on BobaDocs. Click the folder for the guide with the page you want to edit.
```

Comment:
I would add a screenshot of the top bar, and do a guide here similar to what you did further up:

- `engineering`: [what goes here]
- `volunteers`: [what goes here]
...

Reply:
Top bar screenshot is added, but I'm disagreeing on the guide. A few of the requested changes (including the one for the other folder guide) are about moving out-of-scope information somewhere else, and I think explaining what each guide is fits into that. I handled the other one by moving it to a separate docusaurus guide, and linking to that in the section, and am doing the same for the guide about the guides.

If you insist that it should be in the github editing guide too, I'll move it back, but for the next pass/review I want to see if it's okay with that setup.